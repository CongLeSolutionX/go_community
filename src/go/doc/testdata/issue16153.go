// Copyright 2017 The Go Authors. All rights reserved.
// Use of this source code is governed by a BSD-style
// license that can be found in the LICENSE file.

package issue16153

// original test case
const (
	x uint8 = 255
	Y       = 256
)

// variations
const (
	x uint8 = 255
	Y
)

const (
	X int64 = iota
	Y       = 1
)

const (
	X int64 = iota
	Y
)
