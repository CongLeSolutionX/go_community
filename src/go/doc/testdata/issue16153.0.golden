// 
PACKAGE issue16153

IMPORTPATH
	testdata/issue16153

FILENAMES
	testdata/issue16153.go

CONSTANTS
	// 
	const (
		X	int64	= iota
		Y		= 1
	)

	// 
	const (
		X	int64	= iota
		Y
	)

	// original test case 
	const (
		Y = 256
	)

	// variations 
	const (
		Y uint8
	)

